projects/storefrontapp-e2e-cypress/cypress.config.ts

```diff
@@ -14,6 +14,7 @@ export default defineConfig({
     runMode: 2,
   },
   videoUploadOnPasses: false,
+  chromeWebSecurity: false,
```

Comment:
why do we need to set `chromeWebSecurity` to `false`, do we need to remove this line when go to production, if yes do we have a ticket to follow up?

Reply:
it's already in false in pipeline configuration as 

https://github.com/SAP/spartacus/blob/aebb57404ab28761487e90dd904cd1cfed1f281b/projects/storefrontapp-e2e-cypress/cypress.config.ci.ts#L16